oqs/wrapper.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, alg_name):
 
 # The native KEM structure returned by OQS
 class OQS_KEM(ct.Structure):
```

Comment:
Looking at this naming, do we need to call it `OQS_KEM` or could we just call it `KEM`, because there's the `oqs` module's namespacing already?

Reply:
Well spotted! That's keeping in line with Py naming style too.

Reply:
The names are now KeyEncapsulation and Signature for clarity.